content-repo/extra-docs/packs/Malware_Investigation_&_Response.md

```diff
@@ -0,0 +1,163 @@
+| Dynamic Section              | **Netstat Information**      |                   :x:                |              :white_check_mark:                          |               :x:                       |
+
+*Only supports 'file path' / **Only supports 'file name'
+ 
+
+## Endpoint Integrations
+
+- #### [Palo Alto Networks Cortex XDR - Investigation and Response](https://xsoar.pan.dev/marketplace/details/CortexXDR)
+  **Main Playbook**: [Cortex XDR Malware - Investigation And Response](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr-malware---investigation-and-response)
+
+  Noteworthy feature in this pack:
+
+  - **Hunting for Dedicated Insights**
+
+     XDR Insights are suspicious activity detections that occur from a rule and are observed and enriched on a dedicated endpoint. This playbook searches for dedicated Insights by filtering them through Mitre Techniques and the investigated endpoint. 
+
+    Related Playbook: [Cortex XDR - Endpoint Investigation](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr---endpoint-investigation)
+
+- #### [Microsoft Defender For Endpoint](https://xsoar.pan.dev/marketplace/details/MicrosoftDefenderAdvancedThreatProtection)  
+  **Main Playbook**: [MDE Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/mde-malware---investigation-and-response)
+
+  Noteworthy features in this pack:
+  - **Advanced Hunting**  
+   Microsoft Defender For Endpoint provides advanced features to query your integrated instance and search for information. This playbook uses pre-defined commands, enrichment logic, and tagging results as incident evidence. 
+  
+    Related playbook: [MDE - Host Advanced Hunting]()
+           
+  - **Pro-Active Incident Investigation**  
+    The analyst can run the following pro-active incident investigation actions, which can be automated using playbook inputs:
+    - Running an automated investigation
+    - Running antiVirus full scan on the endpoint
+    - Collecting an Investigation Package (forensics). 
+    
+    Related playbook: [MDE - Pro-Active Actions]()
+  
+    **Note**:  
+    Some playbook inputs can be configured on the [MDE Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/mde-malware---investigation-and-response) playbook.
+
+- #### [CrowdStrike Falcon](https://xsoar.pan.dev/marketplace/details/CrowdStrikeFalcon)
+  **Main Playbook**: [CrowdStrike Falcon Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/crowd-strike-falcon-malware---investigation-and-response)
+
+  Noteworthy feature in this pack:
+   - **Allow and Block IOCs Per Device Group**
+
+     Due to the complexity of managing multiple devices and exclusions, a policy may require certain modifications to fit each device group. This playbook enables choosing which device groups (specific or all groups) will be affected by your exclusions.
+
+**Note**:  
```

Comment:
@ssokolovich Deduplication should also be explained more in depth
Like the purpose (why do you need it in the malware context)
Mirroring (when applicable and when not)

Reply:
As for the Dedup logic, we can explain more and add a link to the PB. Is there something specific that you have in mind? 
As for the mirroring- the only place where mirroring is relevant is XDR and CS, but this should be part of the Pack description rather than be described here. Do you think otherwise?

Reply:
Something like

XSOAR supports pre processing rules for incidents as detailed here
https://xsoar.pan.dev/docs/incidents/incident-pre-processing
which can be used for handling incidents which share the same incident fields
However XSOAR also supports deduplication using the Dedup - Generic V4 playbook
Unlike Pre processing rules, deduplication can use a configurable similarity setting and not just exact value match
For for details about the deduplication see
https://xsoar.pan.dev/docs/reference/playbooks/dedup---generic-v4
Some of the deduplication inputs are exposed via the main playbook ......
For other setting detach the vendor specific playbooks and edit

Notice that XDR and CrowdStrike Falcon integrations also support mirroring, meaning that if deduplication closes the XSOAR incident this will also close the incidents on the EDR side